<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prueba</class>
 <widget class="QDialog" name="prueba">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>761</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Prueba de Agilidad</string>
   </property>
   <widget class="QPushButton" name="boton1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../proyecto_alse/RC_led.png</normaloff>
      <normalon>:/images/RC_led.png</normalon>../proyecto_alse/RC_led.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../proyecto_alse/RC_led.png</normaloff>
      <normalon>:/images/RC_led.png</normalon>../proyecto_alse/RC_led.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../proyecto_alse/RC_led.png</normaloff>
      <normalon>:/images/RC_led.png</normalon>../proyecto_alse/RC_led.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../proyecto_alse/RC_led.png</normaloff>
      <normalon>:/images/RC_led.png</normalon>../proyecto_alse/RC_led.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../proyecto_alse/RC_led.png</normaloff>
      <normalon>:/images/RC_led.png</normalon>../proyecto_alse/RC_led.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../proyecto_alse/RC_led.png</normaloff>
      <normalon>:/images/RC_led.png</normalon>../proyecto_alse/RC_led.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../proyecto_alse/RC_led.png</normaloff>
      <normalon>:/images/RC_led.png</normalon>../proyecto_alse/RC_led.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton11">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../proyecto_alse/RC_led.png</normaloff>
      <normalon>:/images/RC_led.png</normalon>../proyecto_alse/RC_led.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton10">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../proyecto_alse/RC_led.png</normaloff>
      <normalon>:/images/RC_led.png</normalon>../proyecto_alse/RC_led.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton9">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../proyecto_alse/RC_led.png</normaloff>
      <normalon>:/images/RC_led.png</normalon>../proyecto_alse/RC_led.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boton8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../proyecto_alse/RC_led.png</normaloff>
      <normalon>:/images/RC_led.png</normalon>../proyecto_alse/RC_led.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Tiempo de prueba:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cont_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>206</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Cantidad de aciertos:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cont"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
